hack/build-images.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script is used build new container images of the CAPV manager and
+# clusterctl. When invoked without arguments, the default behavior is to build
+# a new image for the manager and to tag it as `edge`
+#
+# Options:
+#   -p push the images to gcr.io
+#   -c tag the image as a CI image instead of the default
+#
+# Assumptions:
+#  If using -p to push images, then the env var GCR_KEY_FILE must be set
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# BASE_REPO is the root path of the image repository
+readonly BASE_REPO=gcr.io/cluster-api-provider-vsphere
+
+# CAPV_MANAGER_IMAGE is the path of the CAPV manager container image
+readonly CAPV_MANAGER_IMAGE=${BASE_REPO}/capv-release/manager
+
+# CAPV_MANAGER_IMAGE_CI is the path of the CAPV manager image for CI usage
+readonly CAPV_MANAGER_IMAGE_CI=${BASE_REPO}/capv-ci/manager
```

Comment:
And on using `capv-ci` as a different bucket for CI images?

Reply:
And then for CI images I was thinking `gcr.io/cluster-api-provider-vsphere/ci/xyz`. If you want to keep things in parallel you could use `gcr.io/cluster-api-provider-vsphere/release` for the release images, but I only envisioned this new project as the home for release images at the root of the gcr prefix. I like the long-term thinking though. I guess for release *images* (not other artifacts), I was just thinking that the GCR root itself would be for releases?